sdk/python/feast/repo_config.py

```diff
@@ -113,7 +116,7 @@ class RegistryConfig(FeastBaseModel):
 class RepoConfig(FeastBaseModel):
     """ Repo config. Typically loaded from `feature_store.yaml` """
 
-    registry: Union[StrictStr, RegistryConfig] = "data/registry.db"
+    registry: RegistryConfig = RegistryConfig()
```

Comment:
Unfortunately we can't have a backwards compatible change here. If we introduce this change then all Feast users will have broken deployments. We will need to test and support both the old and new approach (file path, dir path), at least for a few versions to allow gradual upgrades.

Reply:
Ah that makes a lot of sense, let me try to support both versions of the registry config using `Union`. Since I already updated the tests to model the new version, I'll just add back a simple test testing the old version. Does that sound good?

Reply:
I used a `pytest.parametrize` [here in test_cli_local.py](https://github.com/feast-dev/feast/pull/1653/files#diff-ffdc23101f179930d4578bf59d2e2afe6ed94a73fa58368e243c1bee5b252ddaR17) to test both the old registry config and new. There shouldn't be any break changes now, but let me know if you think there should be more robust testing for the old version